dart/example/pubsub/pubsub.dart

```diff
@@ -21,7 +21,7 @@ class PublishService {
 
   // We take a string here, but a message normally just receives bytes.
   Future<void> publish(String message) {
-    stub.publish(PublishRequest(
+    return stub.publish(PublishRequest(
```

Comment:
I will rework this a lot anyway, but no:
since the `publish` function is not marked `async` it's a normal function. It just returns the future from the `stub.publish` call.
Alternatively we could, indeed, do the following:
```
Future<void> publish(String message) async { // <= note the 'async' here.
  await stub.publish(PublishRequest(
```